hydra-main/src/main/java/com/addthis/hydra/job/mq/CommandTaskNew.java

```diff
@@ -16,14 +16,12 @@
 
 public class CommandTaskNew extends AbstractJobMessage {
 
-    private static final long serialVersionUID = 1117109955580588580L;
+    @SuppressWarnings("unused")
+    public CommandTaskNew() {
```

Comment:
constructors that exist only for codec / jackson should be either protected or private. That'll probably also handle the warning.